src/collective/taxonomy/jsonimpl.py

```diff
@@ -24,7 +29,7 @@ def __init__(self, context, request):
         utility_name = request.get('taxonomy', '')
         taxonomy = queryUtility(ITaxonomy, name=utility_name)
         if not taxonomy:
-            raise ValueError('Taxonomy `%s` could not be found.' % utility_name)
+            raise ValueError('Taxonomy `%s` could not be found.' % utility_name)  # noqa
```

Comment:
please don't use generic `noqa` comments; it's better to explicitly specify what are you trying to ignore.